conduct/conduct_faq.md

```diff
@@ -0,0 +1,75 @@
+# Project Jupyter Code of Conduct - Frequently Asked Questions
+
+This FAQ attempts to address common questions and concerns around the Jupyter
+community's Code of Conduct. If you still have questions after reading it,
+please feel free to contact us at
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org).
+
+### Why have you adopted a Code of Conduct?
+
+We think the Jupyter community is awesome. If you're familiar with the Jupyter
+community, you'll probably notice that the Code basically matches what we
+already do. Think of this as documentation: we're taking implicit expectations
+about behavior and making them explicit.
+
+We're doing this because the Jupyter community is growing faster than any of us
+could have anticipated. This is on balance a very positive thing, but as we've
+grown past the point where it's possible to know the whole community, we think
+it's very important to be clear about our values.
+
+We know that the Jupyter community is open, friendly, and welcoming. We want to
+make sure everyone else knows it too.
+
+### What does it mean to "adopt" a Code of Conduct?
+
+For the most part, we don't think it means large changes. We think that the text
+does a really good job describing the way the Jupyter community already conducts
+itself. We expect that most people will simply continue to behave as they have
+in the past.
+
+However, we do expect that people will abide by the spirit and words of the Code
+of Conduct when in "official" Jupyter spaces. This code has been adopted by both
+the Jupyter Steering Council and JupyterDay/JupyterCon events. That means that
+it will apply both in community spaces and at our events.
+
+In practice, this means mailing lists, the various Jupyter channels, and Jupyter
+events.
+
+### What about events funded by Project Jupyter?
+
+This Code of Conduct will also cover any events that Jupyter funds.
+
+The project requires that any events it funds have policies and procedures in
+place for handling harassment. It's especially important to us that real-life
+events take steps to protect the physical and mental security of their
+participants.
+
+Event organizers will enforce the Code of Conduct.
+
+### What happens if someone violates the Code of Conduct?
```

Comment:
I only moved it up one notch b/c the first two items are sort of "introductory questions", and it seems like the reading would break up a bit if I move it any higher.

I've also added language in the main intro about informal self-regulation that ties in to this same idea.  Once I post updates, let me know if that helps...